ntc_templates/templates/ruckus_fastiron_show_version.textfsm

```diff
@@ -0,0 +1,28 @@
+Value List VERSION (\S+)
+Value List BOOTCODE (\S+\s+\S+)
+Value HARDWARE (.*)
+Value List SERIAL (\S+)
+Value List UPTIME (.*)
+
+
+Start
+  ^\s+Copyright\s+\(c\)\s+Ruckus\s+Networks,\s+Inc\.\s+All\s+rights\s+reserved\.
+  ^\s+UNIT\s+[0-9]*:\s+compiled\s+on\s+\S+\s+[0-9]*\s+[0-9]*\s+at\s+[0-9]*:[0-9]*:[0-9]*\s+labeled\s+as\s+\S+
+  ^\s+\([0-9]*\s+bytes\)\s+from\s+Primary\s+\S+\s+\(UFI\)
+  ^\s+SW:\s+Version\s+${VERSION}
+  ^\s+Compressed\s+Primary\s+Boot\s+Code\s+size\s+=\s+[0-9]*,\s+Version:${BOOTCODE}
+  ^\s+Compiled\s+on\s+\S+\s+\S+\s+[0-9]*\s+[0-9]*:[0-9]*:[0-9]*\s+[0-9]*
+  ^\s+HW:\s+${HARDWARE}\s*$$
+  ^UNIT\s+[0-9]*:\s+SL\s+[0-9]*:\s+.*
+  ^\s+Serial\s+#:${SERIAL}
+  ^\s+Software\s+Package:\S+

```

Comment:
You may need to go back through the template to make sure all lines being accounted for are proper.

The matches can be least match as well. So the failing test results at the moment is failing on this line:
`  ^\s+Software\s+Package:\S+`

You can change it to just be: `  ^\s+Software\s+Package` so it minimally matches it without potentially matching other lines as well. There are several lines you can do this on throughout the template.

Reply:
Ok, cool. I made this one a while back, I'll go through and review it.